src/main/kotlin/no/nav/omsorgspenger/App.kt

```diff
@@ -216,6 +220,11 @@ internal class ApplicationContext(
                 saksnummerRepository = benyttetSaksnummerRepository
             )
 
+            val benyttetOverføringerStatistikkService = overføringerStatistikkService ?: OverføringerStatistikkService(
+                    kafkaProducer = benyttetKafkaProducer,
+                    topic = "TODO" // TODO - Legg til riktig topic(er)!!
```

Comment:
hadde jo vært hendig om vi kunne ha en `privat-omsorgspenger-rammemeldinger-statistikk` for alle rammemeldinger. Da kan vi potensielt skille ut en `StatistikkService` utenom overføring som kan brukes med forskjellige parametre fra de forskjellige rammemeldingene. Kanskje `behandlingType` kan brukes til de forskjellige rammemeldingene?